*** datm.buildnml.csh.orig	Fri Jun 25 16:18:13 2010
--- datm.buildnml.csh	Fri Jun 25 16:18:19 2010
***************
*** 32,38 ****
     dataMode       = 'CLMNCEP'
     domainFile     = '$DOMAINFILE'
     streams        = 'clm1PT.1x1_vancouverCAN.stream.txt 1 1992 1992 ',
!                     'presaero.stream.txt 1 2000 2000'
     vectors        = 'null','null'
     mapmask        = 'nomask','nomask'
     tintalgo       = 'linear','linear'
--- 32,38 ----
     dataMode       = 'CLMNCEP'
     domainFile     = '$DOMAINFILE'
     streams        = 'clm1PT.1x1_vancouverCAN.stream.txt 1 1992 1992 ',
!                     'presaero.stream.txt 1 1 1'
     vectors        = 'null','null'
     mapmask        = 'nomask','nomask'
     tintalgo       = 'linear','linear'
***************
*** 119,128 ****
              mask    mask
           </variableNames>
           <filePath>
!             $DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/aero
           </filePath>
           <fileNames>
!             aerosoldep_monthly_1849-2006_1x1_tropicAtl_c091026.nc
           </fileNames>
        </domainInfo>
        <fieldInfo>
--- 119,128 ----
              mask    mask
           </variableNames>
           <filePath>
!             $DIN_LOC_ROOT/lnd/clm2/snicardata
           </filePath>
           <fileNames>
!             aerosoldep_2000_mean_1x1_vancouverCAN_c090716.nc
           </fileNames>
        </domainInfo>
        <fieldInfo>
***************
*** 143,155 ****
              DSTX04DD   dstwet4
           </variableNames>
           <filePath>
!             $DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/aero
           </filePath>
           <offset>
              0
           </offset>
           <fileNames>
!             aerosoldep_monthly_1849-2006_1x1_tropicAtl_c091026.nc
           </fileNames>
        </fieldInfo>
        <!-- Information on the program that created this file -->
--- 143,155 ----
              DSTX04DD   dstwet4
           </variableNames>
           <filePath>
!             $DIN_LOC_ROOT/lnd/clm2/snicardata
           </filePath>
           <offset>
              0
           </offset>
           <fileNames>
!             aerosoldep_2000_mean_1x1_vancouverCAN_c090716.nc
           </fileNames>
        </fieldInfo>
        <!-- Information on the program that created this file -->
